modules/apps/object/object-service/service.xml

```diff
@@ -1200,6 +1200,7 @@
 		<exception>LockedObjectAction</exception>
 		<exception>ObjectActionActive</exception>
 		<exception>ObjectActionConditionExpression</exception>
+		<exception>ObjectActionDescription</exception>
```

Comment:
@Aquiiiles did you run `gw buildService` as I asked [here](https://github.com/liferay-bpm/liferay-portal/pull/4516#discussion_r2086973050)?

 If so, please reorganize your commits.

1. Change the `object-service/service.xml`, by adding the new exception, and add a commit called `LPD-20481 Add new exception`
2. Run `gw buildService` and add a commit called `LPD-20481 buildService` with **only** the automatically generated changes
3. Add a new commit adapting the `ObjectActionDescriptionException` class